SU-HSDO/suhumsci
# Arch 131 style individual person page

# READY FOR REVIEW

# Summary
- Styling the person node page.

# Steps to Test
1. Visit an individual faculty or staff page (which you might have created as part of ARCH-130)
1. Examine the page at all breakpoints
1. Confirm that styling matches comp.

# Associated Issues and/or People
- [JIRA ticket](https://stanfordits.atlassian.net/browse/ARCH-131)
